bin/vscout.dart

```diff
@@ -4,13 +4,16 @@ import 'dart:io';
 
 import 'package:args/command_runner.dart';
 import 'package:io/ansi.dart';
-
 import 'package:vscout_cli/vscout_cli.dart';
 
 // The exit code for a general error.
 int generalError = 1;
 
 main(List<String> args) async {
+  DatabaseHandler databaseHandler = DatabaseHandler();
+  // Create new database handler with empty constructor,
+  await databaseHandler.InitializeDb();
+  // Run all constructor functions in async function so stuff is defined
```

Comment:
Because the constructor function doesnt have a return type and cant "await" future return types, if you try to initialize the db through it, it will just run the function, not wsit for it to finish, and try to call an incomplete db object

Reply:
Good stuff
Write that but a bit more condensed